plugins/fianu/src/api/FianuApiClient.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ConfigApi } from '@backstage/core-plugin-api';
+import { FianuApi } from './FianuApi';
+
+export class FianuApiClient implements FianuApi {
```

Comment:
Not sure that this warrants its own API at the moment given that it only concatenates a string from config which does not need to be async at the moment. Could probably be moved to the component instead.

Reply:
I am planning to add some more functionality in future. so would like to keep it as separate API so we have pattern in place.

Reply:
Sounds good to me 👍